DASH-IF-IOP.bs.md

```diff
@@ -10,7 +10,241 @@ Live discussion in #document-authoring on Slack.
+
+The zero point in the MPD timeline of a dynamic MPD SHALL be mapped to the point in real time indicated by `MPD@availabilityStartTime`. This value SHALL NOT change between MPD updates.
+
+The ultimate purpose of the MPD is to enable the client to obtain media samples for playback. The following data structures are most relevant to locating and scheduling the samples:
+
+1. The MPD consists of consecutive [=periods=] which map data onto the MPD timeline.
+1. Each period contains of one or more [=representations=], each of which provides media samples inside a sequence of [=media segments=].
+1. Representations within a period are grouped in adaptation sets, which associate related representations and decorate them with metadata.
+
+<figure>
+	<img src="Images/Timing/BasicMpdElements.png" />
+	<figcaption>Illustration of the primary elements described by an MPD.</figcaption>
+</figure>
+
+The chapters below explore these relationships in detail.
+
+## Periods ## {#timing-period}
+
+An MPD SHALL define an ordered list of one or more <dfn id="period">periods</dfn>. A period is both a time span on the [=MPD timeline=] and a definition of the data to be presented during the period. Period timing is relative to the zero point of the [=MPD timeline=].
+
+All periods SHALL be consecutive and non-overlapping. In determining the duration of a period, the `Period@start` of the next period, if the attribute is present, SHALL be used to determine the duration of the preceding period, ignoring any conflicting `Period@duration`.
+
+Note: By allowing `Period@start` to override `Period@duration` we enable a simplification often used in live content processing workflows, where a new period "cuts short" an existing period without having to update the MPD elements describing the existing period.
+
+<div class="example">
+The below MPD example consists of two 20-second periods despite the first period having a `Period@duration` value of 30 seconds.
+
+<xmp highlight="xml">
+<MPD type="static" mediaPresentationDuration="PT40S">
+	<Period duration="PT30S">
+		...
+	</Period>
+	<Period start="PT20S" duration="PT20S">
+		...
+	</Period>
+</MPD>
+</xmp>
+</div>
+
+In a static MPD, the first period SHALL start at the zero point of the [=MPD timeline=]. In a dynamic MPD, the first period SHALL start at or after the zero point of the [=MPD timeline=].
+
+In a static MPD, the last period SHALL have a `Period@duration`. In a dynamic MPD, the last period MAY lack a `Period@duration`, in which case it SHALL be considered to have an unlimited duration.
+
+In a static MPD, `MPD@mediaPresentationDuration` SHALL be present. In a dynamic MPD, `MPD@mediaPresentationDuration` MAY be present. When present, `MPD@mediaPresentationDuration` SHALL accurately indicate the duration between the zero point on the [=MPD timeline=] and the end of the last period.
```

Comment:
How about we tighten this all the way to SHALL/SHALL NOT? Or is there any reason to leave out the duration even if you know for sure the manifest will not get any more appends?

Reply:
Using SHALL/SHALL NOT sounds good to me. The **for sure** part of your statement becomes quite important if tightening this. I.e. it's probably not a good idea to write something like:

> SHALL be included when no further content will be appended

Because there may well be a short period of time between the last content being appended and the server knowing for sure that there's not going to be any more. During this period of time `@mediaPresentationDuration` cannnot be added. So it's probably a better idea to write something like:

> SHALL be included when it becomes known that no further content will be appended

Which accounts for this possibility.